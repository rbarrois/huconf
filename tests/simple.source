# Hi
## Some huconf comment
# End

#@if shell
shell-included
#@else
shell-excluded
#@endif

#@if server
server-excluded
#@else
server-included
#@endif
