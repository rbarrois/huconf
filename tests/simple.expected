# Hi
# End

shell-included

server-included
